app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -68,6 +71,36 @@ protected function create(array $data)
             'last_name' => $data['last_name'],
             'email' => $data['email'],
             'password' => bcrypt($data['password']),
+            'email_token' => base64_encode($data['email'])
         ]);
     }
+
+    /**
+     * Handle a registration request for the application.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function register(Request $request)
+    {
+        $this->validator($request->all())->validate();
+        event(new Registered($user = $this->create($request->all())));
+        dispatch(new SendVerificationEmail($user));
+        return view('email.verification');
+    }
+
+    /**
+     * Handle a registration request for the application.
+     *
+     * @param $token
+     * @return \Illuminate\Http\Response
+     */
+    public function verify($token)
+    {
+        $user = User::where('email_token',$token)->first();
+        $user->active = 1;
+        if($user->save()){
```

Comment:
Always add a space after an `if` statement: `if ($user->save()) { ... }`. This are the rules of PSR2, which is good practise to follow. Google it if you don't know what it is :)